.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   extends: 'airbnb',
   rules: {
-    "linebreak-style": 'off', // Airbnb specifies Unix style line endings, turn this off for Windows users
+    'implicit-arrow-linebreak': 'off',
+    'linebreak-style': 'off', // Airbnb specifies Unix style line endings, turn this off for Windows users
```

Comment:
Not related to this PR but I am curious as to why this is switched off, wouldn't this allow different types of line endings to be committed into code if someone using a windows os contributes to the pkg?

Reply:
Because it would have forced me to go through all my code and reformat it. This also might be in conflict with Prettier.